usdigitalresponse/usdr-gost
# Update output file version to 2024-10-01

### Ticket #3561 
## Description
- Ran script to update the output file (instructions here - https://github.com/usdigitalresponse/usdr-gost/blob/main/docs/common-tasks/updating-output-templates.md)

## Screenshots / Demo Video
N/A

## Testing
N/A

### Manual tests for Reviewer

1. Open the app locally
2. Add upload files
3. Hit "Send Treasury Report by Email" button
4. Verify that the email sent contains a zip-file that matches the updated columns

## Checklist
